### DISCUSSION TITLE
Saucer in qmake Qt project

### DISCUSSION THREAD

**Discussion by ank-007** [saucer/saucer#15] [Q&A] [answered]
is it possible to use saucer in Qt project which uses qmake instead of cmake, if yes how to include saucer in .pro and use it in the  application?

**Comment by Curve** [comment, marked as answer]
Saucer is aimed towards usage with CMake, thus CMake is the only build system we support officially.

However, this does not mean it cannot work with other build systems!
One can adapt the build configuration to build systems like qmake but if you want to go down that path we will sadly not be able to help.

**Reply by ank-007** [reply]
thank you for clearing my doubt
